.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225312993-a3d9ff8b-a75d-4eec-8174-0987eaac5d8f.png)

**[Optional]**

- [ ] It would be nice if you make your commit messages captialize to make your pull request professional :+1: